### DISCUSSION TITLE
New Javascript Engine for DotNet

### DISCUSSION THREAD

**Discussion by jogibear9988** [sebastienros/jint#1020] [General]
@lahma @sebastienros 

Today I found this:

https://dev.to/web-atoms/introducing-new-javascript-engine-yantrajs-for-dotnet-49h0

did you know it?

**Comment by lahma** [comment]
This looks interesting, thanks for the heads up! 👍🏻 (converted to discussion to keep issues as work-to-be-done)

**Comment by sebastienros** [comment]
Haven't looked at the code yet, but it seems to be compiling to IL only, which is interesting. I am interested in the perf difference then, to see how much the interpreted mode is slower. Then obviously for single runs compiling will always be slower. Also it's LGPL, not very welcoming but I assume the goal might be to push for the commercial licenses?

**Comment by jogibear9988** [comment]
It could be much worse than LGPL, GPL for example :-)
